tests/test_edf.py

```diff
@@ -1160,3 +1161,43 @@ def test_sampling_frequencies_leading_to_floating_point_issues_in_signal_duratio
     assert edf.num_data_records == 10
     assert edf.signals[0].samples_per_data_record == 22
     assert edf.signals[1].samples_per_data_record == 9
+
+
+# fmt: off
+@pytest.mark.parametrize(
+    ("file_length_in_bytes", "num_records_in_header", "expected_warning"),
+    [
+    #    header   data_length   extra bytes     num records field    expected warning
+        (2048  +  10*18240      +1,             10,                  "Incomplete data record at the end of the EDF file"),
```

Comment:
since `file_length_in_bytes` is the same for all tests, we could remove `2048 + 10*18240` and just pass the value in the column "extra bytes" as `size_delta`

Reply:
Yes, that makes it simpler